.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+<!--
+Thank you for reporting an issue. The more information you can give us, the
+better the chance we can fix your problem.
+
+This issue tracker is for bugs and issues found within node-gyp,
+if you have an issue installing a specific module, please file an issue on
+that module's issue tracker (find the link with `npm issues modulename`).
+
+Please fill in as much of the template below as you're able.
+
+Node Version: output of `node -v` and `npm -v`
+Platform: output of `uname -a` (UNIX), or
+`systeminfo | findstr /B /C:"OS Name" /C:"OS Version" /C:"System Type"` (Windows)
+Compiler: output of `cc -v` (UNIX) or `msbuild /version & cl` (Windows)
+Module: the module you tried to install.
+-->
+
+* **Node Version**:
+* **Platform**:
+* **Compiler**:
```

Comment:
Maybe include how to get the version (`cc -v` and on Windows... `mscver`?   I think?)

Same for platform: `uname -a`